src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
 // write code here
+const ul = document.querySelector('ul');
+const li = document.querySelectorAll('li');
+
+for (const i of li) {
+  const span = document.createElement('span');
+
+  i.prepend(span);
+  span.prepend(span.nextSibling);
```

Comment:
The line `span.prepend(span.nextSibling);` attempts to prepend the next sibling of the span to itself. This is likely incorrect because the span has just been created and does not have a next sibling at this point. Consider revising this logic to correctly append or prepend the desired content to the span.